src/OpenTelemetry.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkInstrumentationEventSource.cs

```diff
@@ -59,6 +69,15 @@ public void InvalidPayload(string handlerName, string eventName)
         this.WriteEvent(4, handlerName, eventName);
     }
 
+    [Event(5, Message = "Enrichment threw exception. Exception {0}.", Level = EventLevel.Error)]
+    private void EnrichmentException(string eventName, string exception)
```

Comment:
CI failed.

> Build error

> D:\a\opentelemetry-dotnet-contrib\opentelemetry-dotnet-contrib\src\OpenTelemetry.Instrumentation.EntityFrameworkCore\Implementation\EntityFrameworkInstrumentationEventSource.cs(85,27): error SA1204: Static members should appear before non-static members [D:\a\opentelemetry-dotnet-contrib\opentelemetry-dotnet-contrib\src\OpenTelemetry.Instrumentation.EntityFrameworkCore\OpenTelemetry.Instrumentation.EntityFrameworkCore.csproj]

I think instead of moving the method it needs to be public.

```suggestion
    [Event(5, Message = "Enrichment threw exception. Exception {0}.", Level = EventLevel.Error)]
    public void EnrichmentException(string eventName, string exception)
```